microcosm_flask/conventions/crud.py

```diff
@@ -34,13 +34,12 @@ def decorator(func):
 
 
 @_crud(Operation.Search)
-def register_search_endpoint(graph, obj, path_prefix, func, request_schema, response_schema):
+def register_search_endpoint(graph, ns, func, request_schema, response_schema):
```

Comment:
All of the internal route registration functions now take a `Namespace`.

This simplifies the call arguments a bit.